README.md

```diff
@@ -19,11 +19,28 @@ The PreliZ documentation can be found in the [official docs](https://preliz.read
 
 ### Last release
 PreliZ is available for installation from [PyPI](https://pypi.org/project/preliz/).
-The latest  version can be installed using pip:
+There are two dependency bundles for the latest version:
 
+* `full`: includes dependencies for all features (interactive and non-interactive)
+* `non-interactive`: includes dependencies for non-interactive features only
+
+And both can be installed using pip:
+
+::::{tab-set}
```

Comment:
tabs do not work in the github readme nor on pypi, so I would not use them here, only in the docs (there is can be used from rst, md or ipynb files, but it is a sphinx feature)